db/schema.rb

```diff
@@ -1630,12 +1630,13 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "version"
-    t.bigint "version_id"
+    t.integer "version_id"
```

Comment:
Are these accidental reversions?  I _think_ it should be `bigint`

Reply:
Yes, accidental, but I'm not sure the cause of this... I don't think this schema matched the migrations.

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/519/files

It's added [here](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/519/files) as an `integer`.  I think the intent of the PR was to add a reference between `institution_archives` and `versions` but the migration doesn't actually do that (but the schema does)

Reply:
Ugh, you're right.  I see what happened.
First, as background, since Rails 5.1, [primary and foreign keys have been `bigint` instead of `integer`](https://github.com/rails/rails/pull/26266)
In the linked PR ([this commit](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/519/commits/dda2012bc87cc043b30c6c075b938cecac37196c)), @elder-joseph added foreign keys to the two 'archive' tables in question.
Then I [approved the PR](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/519#pullrequestreview-337280796).
Then @elder-joseph [changed the migrations to now just add the foreign key field as an `interger`](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/519/commits/b8e7c02818b6bd64fd3256a70e625b16c5be7f66).  I understand why he did this: it has been decided that the 'archive' tables don't need indexes.
The problem is, he then merged the PR without re-running the migrations after the change.
Thus, the schema specifies `bigint` and the migration specifies `integer`.
So when you @dmccurdybah ran the migrations on your machine it changed the field type to `integer` because that's what the migration specifies.
So now we have a potential problem.  Has the migrations in question been run in deployed `dev`, `staging`, and/or `production`? 

**Please**, if you push changes after a PR approval, re-request a review before you merge.

Reply:
Thank, yeah, I understand the issue and this definitely shouldn't have happened.

The migrations haven't been ran on `dev`, `staging` or `production` yet. Is it acceptable to you if we update the existing migrations in a separate PR?

Reply:
Yeah, that's fine with me since we're a small team.  Will just have to make sure all the other engineers are aware so they can ensure that their local databases are accurate.